### PR TITLE
Release notes 1.2.1

### PR DESCRIPTION
**Pull request by KaSroka**

### CODE REVIEW

**Comment by KaSroka** [comment]
Don't merge yet

File: release_notes/nrf_802154_1.2.1.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# nRF IEEE 802.15.4 radio driver 1.2.1 release notes
+
+# TODO
+Release date: 2018-09-??
+# TODO
+
+This is a minor release with bugfixes.
```
Comment (hubertmis) [inline release_notes/nrf_802154_1.2.1.md]: This is not minor release, this is bugfix release. Moreover, I would write something more unique, like "This release fixes build problems in some configurations and allows to run the driver with platforms that modify NVIC settings of the RTC used by the driver (i.e. to manage critical sections)."

File: release_notes/nrf_802154_1.2.1.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* Preserve RTC interrupt status while leaving critical sections. ([PR #155](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/pull/155))
+
+
+## Limitations:
+* Number of supported peers for the Auto Pending Bit feature is limited to 127.
+* (inherited) In multiprotocol configuration (SoftDevice RAAL), the application cannot use IRQ priority levels 2 and 3.
+
+### Limitations of simultaneous operation of SoftDevice Front-End and 802.15.4 Front-End:
+* PPI channels reserved by the SoftDevice (17, 18, 19) should not be used.
+* SoftDevice and 802.15.4 driver should use separate sets of PPI channels, i.e. (1, 2) and (3, 4).
```
Comment (hubertmis) [inline release_notes/nrf_802154_1.2.1.md]: I think 'i.e.' is invalid in this context

File: release_notes/nrf_802154_1.2.1.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+* SoftDevice and 802.15.4 driver should use separate sets of GPIOTE channels, i.e. 4 and (6, 7).
```
Comment (hubertmis) [inline release_notes/nrf_802154_1.2.1.md]: 'i.e.' again

File: release_notes/nrf_802154_1.2.1.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## Known issues:
+* If `NRF_802154_USE_RAW_API` is disabled, a pointer to a frame passed by the transmission result notifications is invalid. Workaround: enable `NRF_802154_USE_RAW_API` if the code relies on the passed pointer.
```
Comment (hubertmis) [inline release_notes/nrf_802154_1.2.1.md]: Both issues are inherited from previous release, should be marked.

File: release_notes/nrf_802154_1.2.1.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+## Bug fixes:
+* Fix build with CSMA-CA disabled. ([PR #154](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/pull/154))
+* Preserve RTC interrupt status while leaving critical sections. ([PR #155](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/pull/155))
+
+
+## Limitations:
+* Number of supported peers for the Auto Pending Bit feature is limited to 127.
+* (inherited) In multiprotocol configuration (SoftDevice RAAL), the application cannot use IRQ priority levels 2 and 3.
+
+### Limitations of simultaneous operation of SoftDevice Front-End and 802.15.4 Front-End:
```
Comment (LuDuda) [inline release_notes/nrf_802154_1.2.1.md]: I would add this description to the FEM section of our wiki. Unless there is a good reason why not, e.g. we plan to fix it.

Comment (hubertmis) [reply release_notes/nrf_802154_1.2.1.md]: Good point. I'll add it to ongoing wiki update.

File: release_notes/nrf_802154_1.2.1.md (hunk 2 above)
Comment (KaSroka) [reply release_notes/nrf_802154_1.2.1.md]: I'm not sure about this, lets wait for TW opinion.

**Comment by b-gent** [comment]
I'm done, please see my comments

**Pull request merged**